recipe/meta.yaml

```diff
@@ -12,10 +12,12 @@ source:
 
 build:
   script: python build.py {{ version }} {{ version_build }} {{ driver_version }}
-  number: 1
+  number: 2
   skip: True  # [win]
 
 requirements:
+  build:
+    - python >=3.6 [osx]  # osx only has python 2 by default
```

Comment:
Selectors need to be behind a comment, ie maye this should be:

```yaml
# osx only has python 2 by default
- python >=3.6  # [osx]
```

Reply:
Ah yes, silly oversight. I had swapped things around from this after the linter had complained.